### ISSUE TITLE
Longer time to recognize in first capture

### ISSUE THREAD

**Issue by Mactacs** [justadudewhohacks/face-api.js#568]
Hi, again @justadudewhohacks :). I've successfully used this API to my projects but my problem now was that it takes a long time to recognize a face in the first capture. It will take about 8 seconds to recognize. But in the next capture it can recognize faces up to 1- 2 seconds. What is the cause of this issue?

**Comment by rjdp** [comment]
Hey @Mactacs did you find a workaround for this ?

**Comment by grghimal** [comment]
@Mactacs @rjdp did you guys find the solution to this problem? I am facing the same problem for the first Recognition. 😢

**Comment by rjdp** [comment]
@grghimal I did not.

**Comment by grghimal** [comment]
@rjdp  thank you for the quick response. Did you continue with the problem or used another library for it?

**Comment by rjdp** [comment]
Yes @grghimal  I had found a very promising lead with a library [MediaPipe](https://github.com/google/mediapipe), check the demo [here](https://codepen.io/mediapipe/pen/dyOzvZM), also check [docs](https://google.github.io/mediapipe/solutions/face_detection.html). Let me know if this works for you

**Comment by grghimal** [comment]
thank you so much @rjdp bhai. I will take a look. Since, This. Face-api Js also works fine for me but, takes around 10 seconds while testing from ipad.
